objects/src/notes/metadata.rs

```diff
@@ -4,23 +4,25 @@ use super::{
     AccountId, ByteReader, ByteWriter, Deserializable, Felt, NoteError, Serializable, Word,
 };
 
-/// Represents metadata associated with a note. This includes the sender, tag, and number of assets.
+// NOTE METADATA
+// ================================================================================================
+
+/// Represents metadata associated with a note.
+///
+/// The metadata consists of:
 /// - sender is the account which created the note.
 /// - tag is a tag which can be used to identify the target account for the note.
```

Comment:
`tag is a tag which can be used to identify the target account for the note.` is this true? 

Isn't the tag used by the target account to identify relevant notes?